src/app/static/src/app/components/header/FileMenu.vue

```diff
@@ -168,22 +138,35 @@
                     }
                 }
             },
+            loadModel() {
+                const input = this.$refs.loadModel as HTMLInputElement;
+                if (input.files && input.files.length > 0) {
+                    const file = input.files[0];
+                    if (this.isGuest) {
+                        const formData = new FormData()
```

Comment:
I have refactored this to use custom getForm. Using file directly from here causes Axios to fail in integration test. I believe it was expecting certain type of format. I tried fixing it but no avail so I ended up using getFormData.

Reply:
Using file directly from the action caused integration test failures? I didn't initially understand why that's a problem since we already had this pattern in the last branch, but doesn't look like we had any load integration tests! Well fair enough, but it seems like a shame to have to structure the code in this way to keep the axios adapter happy. Seems like it ought to mockable somehow. Don't think it's worth holding up this branch over, but could you maybe make a maintenance ticket to get to the bottom of this?

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-6646